src/libraries/Native/AnyOS/System.Security.Cryptography.Native.Browser/pal_crypto_webworker.c

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_browser.h"
+#include "pal_crypto_webworker.h"
+
+// Forward declarations
+extern int32_t dotnet_browser_sha_hash(
+    enum sha_hash ver,
+    uint8_t* input_buffer,
+    int32_t input_len,
+    uint8_t* output_buffer,
+    int32_t output_len);
+
+int32_t SystemCryptoNativeBrowser_SHAHash(
```

Comment:
To avoid confusion of some digest algorithm being added that doesn't use the SHA name, I'd rename all of the "sha_hash" (various spellings thereof) to just "Digest", or "SimpleDigest" (since it's the simple one-shot, vs an accumulator/state-builder)

Reply:
I can do that. The reason I didn't is because SubtleCrypto only supports the SHA family - https://developer.mozilla.org/docs/Web/API/SubtleCrypto/digest. It would also require additional branching when dispatching in JavaScript based on the supplied enum.

Reply:
Done.